app/controllers/admins_controller.rb

```diff
@@ -11,8 +11,9 @@ def new
   def create
     @admin = Admin.new(admin_params)
     @admin.superadmin = false
-    if session[:is_admin] == true and @admin.save
-      #AdminMailer.invite_new_admin(@admin).deliver_now
+    if (session[:is_admin] == true and
```

Comment:
There should not be parenthesis around the condition in a conditional statement per https://codeclimate.com/github/colinschoen/enrollme/pull/12

@raychan3